code/obj/item/organs/flock.dm

```diff
@@ -0,0 +1,24 @@
+/obj/item/organ/flock_crystal
+	name = "jagged crystal"
+	organ_name = "jagged crystal"
+	blood_type = "flockdrone_fluid"
+	blood_color = "#1bdebd"
+	desc = "That thing should not be in there, nopenopenope."
+	icon = 'icons/obj/materials.dmi'
+	icon_state = "starstone" //wooo reused sprites
+	made_from = "gnesis"
+	broken = TRUE
+	unusual = TRUE
+
+	New()
+		..()
+		var/datum/material/M = getMaterial("gnesis")
+		src.setMaterial(M, appearance = TRUE, setname = FALSE)
```

Comment:
we should see if we can make the gnesis material overlay look circuit-y like the doors or something. That'd be neat.

Reply:
currently materials just support colours, but I'm sure that will change with the matsci rework :)

Reply:
materials also support texture overlays. See beewool and pizza for examples I believe

Reply:
#9175